src/player.cpp

```diff
@@ -7833,6 +7824,27 @@ bool player::wield( item& target )
     return true;
 }
 
+bool player::unwield()
+{
+    if( weapon.is_null() ) {
+        return true;
```

Comment:
Discrepancy: wield complains about null item being wielded, but unwield is fine with unwielding a null item.

Reply:
Fixed.